src/statsig/constants.ts

```diff
@@ -27,6 +27,7 @@ export const FeatureGates = {
   [StatsigFeatureGates.SHOW_NFT_REWARD]: false,
   [StatsigFeatureGates.SHOW_JUMPSTART_SEND]: false,
   [StatsigFeatureGates.SHOW_POINTS]: false,
+  [StatsigFeatureGates.SHOW_STABLECOIN_EARN]: false,
```

Comment:
nit on naming: "SHOW_EARN" instead?

Reply:
we talked and debated between the two. show_earn is a bit short, but as long as we continue calling it Earn we should be fine.

Reply:
I think if we anticipate this to be long lived token, we can change to `show_earn`. But almost all of our features have been specific to one release, which is why I went with `show_stablecoin_earn`. So unless we have a case for this being a long lived token (e.g., if we want to have some region lock for this feature), I lean towards `show_stablecoin_earn`.